apps/src/storage/dataBrowser/DataLibraryPane.jsx

```diff
@@ -26,18 +29,32 @@ const styles = {
 };
 
 class DataLibraryPane extends React.Component {
+  static propTypes = {
+    // from redux dispatch
+    onShowWarning: PropTypes.func.isRequired
+  };
+
+  onError = error => {
+    if (
+      typeof error === 'string' &&
+      error.includes('There is already a table with name')
```

Comment:
Perhaps 
'There is already a table with the name "${tableName}"'
so it would display like
There is already a table with the name "cities"

Reply:
Done

Reply:
Did you forget to add the file? :)

Reply:
No, the warning shows the full error message, including the table name.